Examples/Example-iOS/Podfile

```diff
@@ -10,6 +10,6 @@ target 'Example-iOSForPod' do
   # In production, you would use:
   # pod 'GTMAppAuth'
 
-  pod 'AppAuth', '~> 1.0'
+  pod 'AppAuth', '~> 1.6'
```

Comment:
Strictly speaking, the iOS example app itself doesn't need AppAuth 1.6, so I'd leave this unchanged.  Likewise, the macOS example app should be fine where it is.